### ISSUE TITLE
GroupRequiredMixin compatibility with Django 1.11

### ISSUE THREAD

**Issue by kakulukia** [brack3t/django-braces#231]
When using the above Mixin i get the following error
handle_no_permission() takes 1 positional argument but 2 were given
braces/views/_access.py, line 338

Its working when i remove the request parameter, but i dont know what it was used for so i dont feel comfortable fixing it that way. Otherwise if somebody can confirm that this is the right way, i might be able to add a pull request to this ticket.

**Comment by HenryMehta** [comment]
I've also come across this issue.  For me I get this error if the user is not a superuser, but works fine when the user is superuser.

**Comment by HenryMehta** [comment]
Which I have now realised means my previous user didn't have the right Group Permission which is now sorted

**Comment by aastashov** [comment]
> python 2.7
> Django 1.11

Entering a debug of a similar problem found out that inheritance in the class of representations was not used correctly.

this old code
```python
class SomeView(LoginRequiredMixin, SuperuserRequiredMixin, ListView):
    raise_exception = True
    model = SomeModel
```
and i get an error
![image](https://user-images.githubusercontent.com/15646544/68545801-af8a3880-03fa-11ea-9797-8adf496e6800.png)

Then I found SuperuserRequiredMixin and checked the parent method handle_no_permission. He was right. But I checked all my parents' classes for verification and found that LoginRequiredMixin (my first parent) in `handle_no_permission` received only the argument `self`.

I swapped `LoginRequiredMixin` and `SuperuserRequiredMixin`, then this's fixed :D
```python
class SomeView(SuperuserRequiredMixin, LoginRequiredMixin, ListView):
    raise_exception = True
    model = SomeModel
```

I have a poor knowledge of multiple inheritance, but I hope this helps you 😄
